src/pybind/mgr/dashboard/tests/__init__.py

```diff
@@ -140,32 +140,43 @@ def __init__(self, *args, **kwargs):
         PLUGIN_MANAGER.hook.configure_cherrypy(config=cherrypy.config)
         super(ControllerTestCase, self).__init__(*args, **kwargs)
 
-    def _request(self, url, method, data=None, headers=None):
+    def _request(self, url, method, data=None, headers=None, version=None):
```

Comment:
Here, the requests with no data are working fine, but the ones passing some data in the request (_put/_post/_delete) are throwing 500 Internal Server / 415 Content-type error for 'tools.json_in.force': False/True respectively. The reason is the request is receiving empty body even if some data is passed to the request